packages/plugins/documentation/admin/src/pages/Settings.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+
+import { Main } from '@strapi/design-system';
+import { useAPIErrorHandler, Page, useNotification } from '@strapi/strapi/admin';
+import { useIntl } from 'react-intl';
+
+import { SettingsForm } from '../components/SettingsForm';
+import { useGetInfosQuery, useUpdateSettingsMutation } from '../services/api';
+import { getTrad } from '../utils';
+
+const SettingsPage = () => {
+  const { toggleNotification } = useNotification();
+  const { formatMessage } = useIntl();
+  const { formatAPIError } = useAPIErrorHandler();
```

Comment:
I think this should be using the _unstableFormatAPIError property, I don't think the errors are handled correctly if we use this one with Redux Toolkit's base query.

Reply:
ho didn't know about that one ty